api/src/main/resources/messages.properties

```diff
@@ -494,10 +494,10 @@ coreapps.units.meters=m
 coreapps.units.centimeters=cm
 coreapps.units.millimeters=mm
 coreapps.units.kilograms=kg
-coreapps.units.degreesCelsius=°C
+coreapps.units.degreesCelsius=�C
 coreapps.units.perMinute=/min
 coreapps.units.percent=%
-coreapps.units.degreesFahrenheit=°F
```

Comment:
What change is this?

Reply:
i seem not able to see the actual change , its neither a space nor a new line , cant see the actual change